src/reference/asciidoc/amqp.adoc

```diff
@@ -4460,7 +4460,36 @@ You can also use a properties bean to set the property globally for all containe
 [source,xml]
 ----
 <util:properties id="spring.amqp.global.properties">
-    <prop key="smlc.missing.queues.fatal">false</prop>
+    <prop key="mlc.missing.queues.fatal">false</prop>
```

Comment:
Need to document this change in what's new and/or still recognize the old property for backwards compatibility.